main.js

```diff
@@ -608,10 +608,15 @@ function storeUser(id, name) {
 
 function processTelegramText(msg) {
     var now = new Date().getTime();
-
-    // ignore all messages older than 30 seconds
-    if (now - msg.date * 1000 > 30000) {
-        adapter.log.warn('Message from ' + msg.from.name + ' ignored, becasue too old: ' + msg.text);
+	var pollingInterval = 0;
+	if (adapter.config && adapter.config.pollingInterval !== undefined)
+	{
+		pollingInterval = parseFloat(adapter.config.pollingInterval);
```

Comment:
Better use parseInt here because you do not want to have it as "Float" value :-)

Reply:
pollingInterval = parseInt(adapter.config.pollingInterval, 10) || 0

adapter.config is always valid